api-codegen/pom.xml

```diff
@@ -88,7 +87,7 @@ https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codeg
         <dependency>
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
-            <classifier>no-tzdb</classifier>
+            <classifier>no-tzdb</classifier> 
```

Comment:
This version of joda-time doesn't have all the time zone information, I'm not sure what the ramifications are if this artifact is used alone. The integration tests may need to override this, or add the DB separately. Although, it'd probably be better to just change it here to simplify things for consumers, since it's no longer needed for Android. Originally, I put this here because Android needs the no-tzdb build, but I am excluding the joda-time transitive dependency altogether.